sourcegraph/sourcegraph
# web: drop `bootstrap` depenedency

## Context

This PR removes `bootstrap` from `package.json` dependencies and updates documentation to reflect that we do not use `bootstrap` directly anymore.

Closes #29200

## Test plan

1. Check out the updated documentation.
2. Ensure that CI is green.
3. Ensure that `bootstrap` is not present in the lock file.

## App preview:

- [Web](https://sg-web-vb-drop-bootstrap-dependency.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-iuqeffhxxu.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
